Polyhedron/demo/Polyhedron/Plugins/Surface_mesh_deformation/Scene_edit_polyhedron_item.cpp

```diff
@@ -1104,7 +1217,7 @@ void Scene_edit_polyhedron_item::selected(const std::set<Polyhedron::Vertex_hand
 bool Scene_edit_polyhedron_item::insert_control_vertex(vertex_descriptor v)
 {
   if(!is_there_any_ctrl_vertices_group()) {
-    print_message("There is no group of control vertices, create one!");
+    std::cerr<<"There is no group of control vertices, create one!\n";
```

Comment:
Because the fonction print_message() does strickly nothing. It is supposed to do a cout but instead it is commented.